docs/install.rst

```diff
@@ -155,19 +164,35 @@ unless you are fully aware of the risks.
 Using Conda
 -----------
 
+To install ``astropy`` using conda run::
+
+    conda install astropy
+
 ``astropy`` is installed by default with the `Anaconda Distribution
 <https://www.anaconda.com/distribution/>`_. To update to the latest version run::
 
     conda update astropy
 
 There may be a delay of a day or two between when a new version of ``astropy``
-is released and when a package is available for Anaconda. You can check
+is released and when a package is available for conda. You can check
 for the list of available versions with ``conda search astropy``.
 
+It is highly recommended that you install all of the optional dependencies with::
+
+    conda install -c astropy -c defaults \
```

Comment:
if -c defaults actually needed?

Reply:
Also as an aside, this is why we need conda to support optional extra dependencies. See https://github.com/conda/conda/issues/7502 for a possible approach (using 'outputs'). We should really just be able to do ``conda install -c astropy astropy.all`` or something like that.

Reply:
I worry that users will set up their `.condarc` in a way that might prefer other channels in particular conda-forge.  I think it is relatively easy to accidentally get in that situation and start pulling core packages from conda-forge instead of defaults.

Reply:
As a specific example, if a user does this seemingly innocuous command to add conda-forge to channels, then bad things happen:
```
conda config --add channels conda-forge
conda update jplephem
The following packages will be SUPERSEDED by a higher-priority channel:

  attrs                                           pkgs/main --> conda-forge
  certifi              pkgs/main::certifi-2020.4.5.1-py37_0 --> conda-forge::certifi-2020.4.5.1-py37hc8dfbb8_0
  importlib-metadata pkgs/main::importlib-metadata-1.6.0-p~ --> conda-forge::importlib-metadata-1.6.0-py37hc8dfbb8_0
  importlib_metadata                              pkgs/main --> conda-forge
  jplephem                                          astropy --> conda-forge
  kiwisolver         pkgs/main::kiwisolver-1.2.0-py37h04f5~ --> conda-forge::kiwisolver-1.2.0-py37ha1cc60f_0
  libgfortran        pkgs/main::libgfortran-3.0.1-h93005f0~ --> conda-forge::libgfortran-3.0.1-0
  libiconv              pkgs/main::libiconv-1.16-h1de35cc_0 --> conda-forge::libiconv-1.15-h0b31af3_1006
  mkl-service        pkgs/main::mkl-service-2.3.0-py37hfbe~ --> conda-forge::mkl-service-2.3.0-py37h0b31af3_0
  ncurses                 pkgs/main::ncurses-6.2-h0a44026_1 --> conda-forge::ncurses-6.1-h0a44026_1002
  openssl              pkgs/main::openssl-1.1.1g-h1de35cc_0 --> conda-forge::openssl-1.1.1g-h0b31af3_0
  py                                              pkgs/main --> conda-forge
  pyparsing                 pkgs/main::pyparsing-2.4.7-py_0 --> conda-forge::pyparsing-2.4.7-pyh9f0ad1d_0
  pytest-arraydiff                                  astropy --> conda-forge
  pytest-astropy-he~                                astropy --> conda-forge
  pytest-doctestplus                                astropy --> conda-forge
  pytest-openfiles                                  astropy --> conda-forge
  pytest-remotedata                                 astropy --> conda-forge
  python             pkgs/main::python-3.7.7-hc70fcce_0_cp~ --> conda-forge::python-3.7.6-cpython_h1fd5dd1_6
  python-dateutil                                 pkgs/main --> conda-forge
  readline               pkgs/main::readline-8.0-h1de35cc_0 --> conda-forge::readline-8.0-hcfe32e1_0
  soupsieve          pkgs/main/noarch::soupsieve-2.0.1-py_0 --> conda-forge/osx-64::soupsieve-2.0.1-py37hc8dfbb8_0
  sqlite                pkgs/main::sqlite-3.31.1-h5c1f38d_1 --> conda-forge::sqlite-3.30.1-h93121df_0
  tornado            pkgs/main::tornado-6.0.4-py37h1de35cc~ --> conda-forge::tornado-6.0.4-py37h9bfed18_1
  webencodings       pkgs/main/osx-64::webencodings-0.5.1-~ --> conda-forge/noarch::webencodings-0.5.1-py_1
  xz                         pkgs/main::xz-5.2.5-h1de35cc_0 --> conda-forge::xz-5.2.5-h0b31af3_0
  zipp                                            pkgs/main --> conda-forge
  zstd                     pkgs/main::zstd-1.4.4-h1990bb4_3 --> conda-forge::zstd-1.4.4-h4b3e974_3
```